cli/compose/loader/merge.go

```diff
@@ -0,0 +1,107 @@
+package loader
+
+import (
+	"reflect"
+	"sort"
+
+	"github.com/docker/cli/cli/compose/types"
+	"github.com/imdario/mergo"
+	"github.com/pkg/errors"
+)
+
+type specials struct {
+	m map[reflect.Type]func(dst, src reflect.Value) error
+}
+
+func (s *specials) Transformer(t reflect.Type) func(dst, src reflect.Value) error {
+	if fn, ok := s.m[t]; ok {
```

Comment:
Why don't you just return s.m[t]?
```go
func (s *specials) Transformer(t reflect.Type) func(dst, src reflect.Value) error {
    return s.m[t]
}
```
Isn't it equivalent?

Reply:
It would be :+1: